src/fluids.md

```diff
@@ -0,0 +1,184 @@
+# Navier-Stokes Mini Application
+
+The solver implemented in this miniapp solves the transient incompressible
+Navier-Stokes equations.
+
+## Theory
+
+The equations are given in the non-dimensionalized form
+
+\begin{align}
+    \frac{\partial u}{\partial t} + (u \cdot \nabla) u - \frac{1}{Re} \nabla^2 u + \nabla p &= f & \quad \text{in } \Omega\\\\
+    \nabla \cdot u &= 0 & \quad \text{in } \Omega
+\end{align}
+
+where $Re$ represents the Reynolds number. In order to solve these equations,
+the method presented in Tomboulides (1997)[^1] is used, which is based on an
+equal order finite element discretization on quadrilateral or hexahedral
+elements of high polynomial order. The method describes an implicit-explicit
+time-integration scheme for the viscous and convective terms respectively.
+Introducing the following notation for the linear term $L(u) = \frac{1}{Re}
+\nabla u$ and the nonlinear term $N(u) = -(u \cdot \nabla) u$ and their
+time-extrapolated forms
+
+\begin{align}
+    L^\*(u^{n+1}) = \sum_{j=1}^k a_j L(u^{n+1-j}) \\\\
```

Comment:
I don't see L^\* used in the subsequent equations.  This could indicate a typo in the following equations or that this definition is unnecessary.

Reply:
It is used in the form of L^*_x in (9) and (10)

Reply:
Yes, but that's a different function.  You could also make the argument that we don't need to define L^*_x because it's definition is implied by that of N^*.  I think it may be better to define L^*_x after defining L_x and drop the definition of L^*.

Reply:
I tried to make it more clear. I see the issue with the previous version.